packages/components/psammead-navigation/src/hooks/useMediaQuery.js

```diff
@@ -1,18 +1,18 @@
 import { useState, useEffect } from 'react';
 
 const useMediaQuery = query => {
-  const isClient = typeof window !== 'undefined';
+  const [matches, setMatches] = useState(null);
```

Comment:
A brief explanation into why this change was made, but I can explain in person on Monday too. Previously, this hook was returning `undefined` on the server render, and `true` or `false` on the initial client render. This meant that the `tabIndex` for navigation links was being set to `null` on the server, and `-1` on the client - so there was a slight difference in markup. This confused react during hydration, so the `tabIndex` was not set to -1, as it needed to be. This change means that when the component initially mounts on the client, matches is still `null`, and when `useEffect` runs on the client, it updates the value to either `true` or `false`. This was the desired effect with the original code, but this ensures that the server and initial client markup matches.

Reply:
Thanks for the explanation @tochwill :)